### PR TITLE
switch pages

### PR DESCRIPTION
**Pull request by VanessaDiBella**

### CODE REVIEW

File: components/HomeLink/index.js [hunk 1]
```diff
@@ -17,5 +17,5 @@ const StyledLink = styled(Link)`
 `;
 
 export default function HomeLink() {
-  return <StyledLink href="/">Start</StyledLink>;
+  return <StyledLink href="/exercises">Start</StyledLink>;
```
Comment (Roland-Hufnagel) [inline components/HomeLink/index.js]: Your wording is confusing me. What is the 'homepage' in your eyes? In your userstory you say: 
"/" URL goes to the homepage
 "/exercise" URL goes to the exercise page
When you create a component 'HomeLink' I expect it to lead to the homepage, not to '/exercises'.
(Typically the 'homepage' is the landing page when opening the base url, which means "/")

File: pages/index.js [hunk 2]
```diff
@@ -1,108 +1,33 @@
...
-      <ExercisesList exercises={filteredExercises} />
-    </StyledSection>
-  );
-}
+const HomeBody = styled.section`
+  background-color: #fac74e;
+  min-width: 100%;
+  min-height: 100%;
+  position: absolute;
+  box-sizing: border-box;
```
Comment (Roland-Hufnagel) [inline pages/index.js]: Is there a need to redefine it? You have box-sizing: border-box already defined in you styles.js

File: pages/index.js [hunk 3]
```diff
@@ -1,108 +1,33 @@
...
-text-align: left;
-margin: 0.5rem 0;
-padding: 0.25rem 0.5rem;
-border-radius: 10px;
-align-items: center;
-`;
\ No newline at end of file
+export default function HomePage() {
+  return (
+    <>
```
Comment (Roland-Hufnagel) [inline pages/index.js]: You don't need fragments.

File: pages/index.js [hunk 4]
```diff
@@ -1,108 +1,33 @@
...
+      <HomeBody>
+        <HomeSection>
```
Comment (Roland-Hufnagel) [inline pages/index.js]: Why do you nest a section in a section? One section would be enough. You can define all stylings there.

**Review by Roland-Hufnagel — CHANGES_REQUESTED** [review]
Hi team. Please have a look at my comments. There are a few points you can improve.

File: components/HomeLink/index.js (hunk 1 above)
Comment (VanessaDiBella) [reply components/HomeLink/index.js]: I renamed the components to be more specific.
We want to have a "home" page or more likely a "start" page, that shows up with the app name and purpose etc. which will be in the future a page to log in or register for a personal account.  Thats why we wanted this page as the first page "/"

**Review by Roland-Hufnagel — APPROVED** [review]
This looks better now 👍

**Pull request merged**